### ISSUE TITLE
Significant performance degradation

### ISSUE THREAD

**Issue by CyaNox** [TeamOpenIndustry/ImmersiveRailroading#686]
**Describe the bug**  
Fairly significant performance degradation in comparison to 1.6.1

**Screenshots**  
**1.6.1**:
![javaw_2019-11-12_15-54-40](https://user-images.githubusercontent.com/211381/68683804-b6828980-0567-11ea-88c0-fa5c891c2663.png)

**1.7.1**"
![javaw_2019-11-12_16-07-56](https://user-images.githubusercontent.com/211381/68683820-bc786a80-0567-11ea-9e05-cb5876008bac.png)

**System Information**  
OS: Windows
System Memory: 8GB
Allocated Memory: 4616 MB
CPU: AMD Phenom II X4 955 Black 3.2Ghz
GPU: AMD Radeon HD 6670 1GB
IR-Version: 1.7.1_1.12.2
Forge-Version:  14.23.5.2847

**Modlist**  
https://paste.dimdev.org/oqugigogun.txt
 
**latest.log**
https://paste.dimdev.org/fakehifuwe.log

**Additional context**
I am running a resource pack that reduces some of the texture sizes significantly but it was made for 1.6.1 and not updated yet to 1.7.1 but I don't think it matters too much since the new textures aren't in the world yet but I could be mistaken so here is the resource pack: http://cyanox.nl/IR_lowres.zip

**Comment by cam72cam** [comment]
Try to close some background programs.  Your machine is already out of GPU memory by the time that MC is loading:
```
[14:44:49] [Client thread/INFO] [universalmodcore]: GPU Memory: 100% 1024/1024MB
```

**Comment by CyaNox** [comment]
That is normal ... it also happens in 1.6.1 and no other apps use the GPU. Also it goes far beyond 1024GB due to shared memory. 1.6.1 easily goes up to 1.25GB even though I only have 1GB as shown in ProcessExplorer.

I'll have a more detailed comparison between the two tomorrow as there is definitely a major difference in performance.

**Comment by cam72cam** [comment]
You need to figure out what's using all of your GPU memory.  that's what's causing the issue here.

**Comment by CyaNox** [comment]
After some additional investigation I found that my low resolution resource pack no longer seems to work. This is thus the cause of a lot more GPU memory usage in 1.7.1.

My understanding of resourcepacks and how they interact with Immersive Railroading is limited but from what I understood I could simply shadow all the textures in the `assets` folder as seen in the jar file and this would mostly work. I did notice that not all textures worked but enough worked that it significantly reduced GPU memory on my system.

I'm curious if I should make a new issue with regards resourcepack texture override support (or lack thereof) or if it is okay within this issue?
